components/dashboard/src/projects/NewProject.tsx

```diff
@@ -266,7 +234,15 @@ export default function NewProject() {
             <span className={"pl-2 text-gray-600 dark:text-gray-100 text-base " + (addClasses || "")}>{label}</span>
         </div>)
         const result: ContextMenuEntry[] = [];
-        for (const [ account, props ] of accounts.entries()) {
+
+        if (!selectedAccount && user && user.name && user.avatarUrl) {
+            result.push({
+                title: "user",
+                customContent: renderItemContent(user?.name, user?.avatarUrl),
+                separator: true,
+            })
+        }
```

Comment:
**Thought**: This is definitely out of scope for this code review, but I wonder if maybe we shouldn't have the GitHub user in the list all the time, and not just when there is no `selectedAccount`?

The rationale goes like this:

| 1. First time seeing this page | 2. After installing app for an org | 3. After installing app for your user |
| --- | --- | --- |
| <img width="401" alt="Screenshot 2021-12-15 at 11 13 38" src="https://user-images.githubusercontent.com/599268/146168469-5b5b7017-8e1d-479d-a8f5-c3fca100fb34.png"> | <img width="434" alt="Screenshot 2021-12-15 at 11 14 13" src="https://user-images.githubusercontent.com/599268/146168509-4ae60cb7-1d1a-4733-a90f-794fe679c0d9.png"> | <img width="426" alt="Screenshot 2021-12-15 at 11 14 47" src="https://user-images.githubusercontent.com/599268/146168536-263d2640-9047-4eff-be9d-d295118d15bf.png"> |
| Cool! I see my user 👍 | Oh no! My user is gone? | Aah, my user is back 😌 |

Adding the user in case 2 (i.e. there is a selected account, but the user is still not in the list of accounts) might help.

I think this could be achieved by replacing the `!selectedAccount` condition by something that checks for whether `user` is actually in the list of `accounts`.

However, this should probably be a follow-up issue / PR. 😇

Reply:
You're right - let's discuss this in a follow up PR, as we also have follow-up tasks for the same issue 🙌🏽